src/Amazon.ElasticBeanstalk.Tools/Commands/CreatePackageCommand.cs

```diff
@@ -0,0 +1,203 @@
+﻿using Amazon.Common.DotNetCli.Tools;
+using Amazon.Common.DotNetCli.Tools.Options;
+using Amazon.ElasticBeanstalk.Model;
+using Amazon.S3.Transfer;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using ThirdParty.Json.LitJson;
+
+namespace Amazon.ElasticBeanstalk.Tools.Commands
+{
+    public class CreatePackageCommand : EBBaseCommand
+    {
+        public const string COMMAND_NAME = "create-package";
+        public const string COMMAND_DESCRIPTION = "Package the application to zip file to be deployed later";
+
+        public static readonly IList<CommandOption> CommandOptions = BuildLineOptions(new List<CommandOption>
```

Comment:
Filter this list to the switches that are actually used.

Reply:
done in the last commit